packages/request-client.js/src/api/request-network.ts

```diff
@@ -138,9 +140,13 @@ export default class RequestNetwork {
    * Create a Request instance from an existing Request's ID
    *
    * @param requestId The ID of the Request
+   * @param disablePaymentDetection if true, skip the payment detection
    * @returns the Request
    */
-  public async fromRequestId(requestId: RequestLogicTypes.RequestId): Promise<Request> {
+  public async fromRequestId(
+    requestId: RequestLogicTypes.RequestId,
+    disablePaymentDetection: boolean = false,
```

Comment:
for this one and all others: would it be more flexible having an `options` object instead of properties? we might have other options in the future...
```
fromRequestId(
    requestId: RequestLogicTypes.RequestId,
    options: {    disablePaymentDetection: boolean },
)
```